compiler/dyno/tools/chpldoc/arg-helpers.cpp

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright 2022 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "arg-helpers.h"
+#include "chpl/util/filesystem.h"
+
+bool developer = false;
+
+
+bool startsWith(const char* str, const char* prefix) {
+  return (0 == strncmp(str, prefix, strlen(prefix)));
+}
+
+void clean_exit(int status) {
+  exit(status);
+}
+
+void setupError(std::string msg) {
+  printf("setupError %s", msg.c_str());
```

Comment:
I can't imagine this is particularly useful.
Can you change the `INT_FATAL`s into `assert` calls?

Reply:
Actually, I can't see how this can pass testing, because `USR_FATAL` and `USR_FATAL_CONT` will run it.